runtime/bootstrapper.ts

```diff
@@ -15,6 +15,9 @@ export class SkyAppBootstrapper {
 
   public static config: SkyuxConfig;
 
+  /* istanbul ignore next */
+  constructor() { }
```

Comment:
Istanbul still has difficulty covering non-decorated classes. This will get this file's coverage up to 100%.

Reply:
What does "non-decorated" classes mean?  Is that going to be a common scenario for our consumers?  The reason for this PR is to fix that problem.

If it really isn't common, do we need the empty constructor here?

Reply:
Good catch.